driver/flags_table.c

```diff
@@ -646,6 +646,11 @@ const struct ppm_name_value epoll_create1_flags[] = {
 	{0, 0},
 };
 
+const struct ppm_name_value machine_info_flags[] = {
+	{"BPF_STATS_ENABLED", PPM_BPF_STATS_ENABLED},
+	{0, 0},
+};
```

Comment:
I'm not entirely sure where this is coming from, it's not in our master or the upstream master, think it might be an old change that was removed upstream and somehow crawled its way back in.

Reply:
removed